### ISSUE TITLE
Site Search - Default results, site-docs indexing

### ISSUE THREAD

**Issue by 4down** [cppalliance/temp-site#738]
Is there a way to have the default search results (when no keyword is entered) be an explicitly-defined set of pages rather than the results it is giving now?  Right now it has one result whose body text only shows as “]]” (Chrono) and then near 30 different results for Any.  Even if those are the "proper" results for an empty string search, I don't think there's much use to a user to get those results.

Also, is it indexing the site docs?  I tried searching for “Explore the Content” but nothing comes up.  (https://www.stage.boost.cppalliance.org/doc/user-guide/explore-the-content.html) 

Thanks!

**Comment by ashtum** [comment]
> Is there a way to have the default search results (when no keyword is entered) be an explicitly-defined set of pages rather than the results it is giving now? Right now it has one result whose body text only shows as “]]” (Chrono) and then near 30 different results for Any. Even if those are the "proper" results for an empty string search, I don't think there's much use to a user to get those results.

Yeah, I think the straightforward solution would be to avoid performing any query with an empty string.

> Also, is it indexing the site docs? I tried searching for “Explore the Content” but nothing comes up. (https://www.stage.boost.cppalliance.org/doc/user-guide/explore-the-content.html)
> 

No, it currently only searches in library documentations. I'll add other sections like docs, news, and mailing lists in the future.

**Comment by ashtum** [comment]
The issue of showing results for empty search query has been resolved in this PR: https://github.com/cppalliance/boost-gecko/pull/142.
